manifests/ops-files/change-audit-log.yml

```diff
@@ -0,0 +1,13 @@
+- type: replace
+  path: /instance_groups/name=master/jobs/name=kube-apiserver/properties/k8s-args/audit-log-maxage
+  value: ((k8s-args-audit-log-maxage))
```

Comment:
Not sure that the variable needs to be this verbose. `k8s-args-` isn't necessarily a meaningful prefix since that set of properties exists for a number of jobs. Might just shrink this (and the other variables) to `audit-log-maxage`.

Reply:
Ok I'll do that